R/screenshot.R

```diff
@@ -0,0 +1,60 @@
+#' A function to make screenshots from an OTTR bookdown website
+#' @description This function creates screenshots of course chapters that are stored in a created output directory 
+#' @param git_pat default is NULL; required argument; a Git secret
+#' @param repo default is NULL; required argument; GitHub repository name, e.g., jhudsl/OTTR_Template
+#' @param output_dir default is "resources/chapt_screen_images"; Output directory where the chapter's screen images should be stored
+#' @param base_url default is NULL; rendered bookdown URL where screenshots are taken from, if NULL, the function will use the repo_name and and git_pat to find the base_url
+#' @import cow
+#' @import dplyr
+#' @importFrom webshot2 webshot
+#' @importFrom magrittr %>%
+#' @importFrom rprojroot find_root has_dir
+#' @author Candace Savonen
+make_screenshots <- function(git_pat = NULL, repo = NULL, output_dir = "resources/chapt_screen_images", base_url = NULL){
+  
+  # Find .git root directory
+  root_dir <- find_root(has_dir(".git"))
+  
+  output_folder <- file.path(output_dir)
+  if (!dir.exists(output_folder)) {
+    dir.create(output_folder, recursive = TRUE)
+  }
+  
+  if (is.null(base_url)){
+    base_url <- cow::get_pages_url(repo_name = repo, git_pat = git_pat) #what if these arguments are still NULL/not supplied?
```

Comment:
Do we want to make this cow function part of ottrpal?

Reply:
Yes! That's next up! It's mentioned in #137 but I didn't realize I didn't make it its own issue so there is one now: https://github.com/jhudsl/cow/issues/27